docs/supported.md

```diff
@@ -100,7 +100,7 @@ Earlier and Later versions might be only partially supported, we don't fully kno
 
 Incremental Static Regeneration (ISR) is a rendering mode in Next.js that allows you to automatically cache and periodically regenerate pages with fresh data. Next.js [does not support](https://nextjs.org/docs/pages/building-your-application/rendering/incremental-static-regeneration) building ISR pages for the edge runtime, and as such, pages should be changed to use server side rendering (SSR) instead.
 
-ISR pages are built by the Vercel CLI to something called prerender functions. These are Node.js Lambda functions that can be called in the background while serving the page from the cache. It is not possible to use these with Cloudflare Pages and they are not compatible with the edge runtime.
+ISR pages are built by the Vercel CLI to generate Vercel Serverless Functions. These are Node.js Lambda functions that can be called in the background while serving the page from the cache. It is not possible to use these with Cloudflare Pages and they are not compatible with the [edge runtime](https://nextjs.org/docs/app/api-reference/edge) currently.
```

Comment:
I'm not sure I agree with this change. I specifically referred to them as prerender functions here because there are two types of serverless functions, and ISR uses the prerender variant.
- [Prerender functions](https://vercel.com/docs/build-output-api/v3/primitives#prerender-functions) that come with a prerender config and fallback file.
- [Serverless functions](https://vercel.com/docs/build-output-api/v3/primitives#serverless-functions) that are always run.

Another distinction relevant to this library is that in the case of prerender functions, we use the fallback files, but with regular serverless functions, we error during the build.

Perhaps it would be better if we called them Prerender functions but included a link to the Vercel docs on them?

```suggestion
ISR pages are built by the Vercel CLI to generate Vercel [Prerender Functions](https://vercel.com/docs/build-output-api/v3/primitives#prerender-functions). These are Node.js Lambda functions that can be called in the background while serving the page from the cache. It is not possible to use these with Cloudflare Pages and they are not compatible with the [edge runtime](https://nextjs.org/docs/app/api-reference/edge) currently.
```